Update README.md with more questions than answers (#179)
unicode-org/cldr-staging: README.md

![maven](https://github.com/unicode-org/cldr-staging/workflows/maven/badge.svg)

> It also contains charts.

Reviewer: ```suggestion
It also contains charts, [the HTML website summarizing the CLDR data](https://www.unicode.org/cldr/charts/latest). Read more about what the charts are in [the unicode documentation](https://cldr.unicode.org/index/charts)
```

> ## I have questions. What even is production data, where does it come from, why does this repo exist?

Reviewer: I like the fun tone but I think we should be a little more serious.

```suggestion
## What is production data, where does it come from, why does this repo exist?
```

- Production data means what you get from downloads at <https://cldr.unicode.org/index/downloads>

> - The [inheritance marker](https://unicode.org/reports/tr35/47/tr35.html#inheritance-marker) is in the source git repo, but not in the production data.

Reviewer: ```suggestion
- The [inheritance marker](https://unicode.org/reports/tr35/47/tr35.html#inheritance-marker)(`↑↑↑`) is in the source git repo. It is used to indicate that the data for a locale comes from its parent locale or language. The inheritance markers are replaced by the actual values in the production data.
```

- The tool GenerateProductionData can generate production data (hence the name) from the source repo.
- We use this **cldr-staging** repo to contain a copy of the production data.

> ## Does this repo have anything to do with the fabled cldr-staging survey tool instance?
> 
> - No. Next question.

Reviewer: ```suggestion
## Is this repository used by the Survey Tool cldr-staging instance?

The [Survey Tool](https://cldr.unicode.org/index/survey-tool) is unrelated.
```
